src/webgl/texture.js

```diff
@@ -608,6 +626,21 @@ export default class Texture extends Resource {
   }
   */
 
+  // OVERRIDES
+  setParameters(parameters) {
```

Comment:
Are these deprecated options present in `opts.parameters` or in `opts` (in constructor), if it is in `opts`,  I may have to extract them , add it to `parameters` before calling `setParameters` in constructor ?